code/modules/cargo/packs/security.dm

```diff
@@ -52,6 +52,14 @@
 					/obj/effect/spawner/lootdrop/f13/attachments,
 					/obj/effect/spawner/lootdrop/f13/gunparts/tier3)
 
+/datum/supply_pack/security/binocularsf13
```

Comment:
Maybe adjust the name and include map tools, as well?

Reply:
I guess I can add some mortal shells as well, then.